draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -302,7 +302,7 @@ white space, encoding of Unicode characters, and ordering of object properties
 are allowed. For example, the following strings are all valid and encode the
 same claim value "Möbius":
 
- * A different way to encode the umlaut (two dots `¨` placed over the letter):\
+ * A different way to encode the unicode umlaut:\
```

Comment:
```suggestion
 * A different way to encode the unicode umlaut (two dots `¨` placed over the letter):\
```
for non english speakers, explanation of what umlaut is, is pretty useful..

Reply:
the  "\`¨\`" is the source of the `&#168` issue though...

Reply:
ah right..